announcements/_posts/2022-04-13-pyhep2022.md

```diff
@@ -0,0 +1,9 @@
+---
+title: PyHEP 2022 "Python in HEP" (virtual) Workshop, 12-16 September 2022
+author: Eduardo Rodrigues
+layout: newsletter
+symbol: glyphicon-calendar
+link: https://indico.cern.ch/e/pyhep2021
```

Comment:
```suggestion
link: https://indico.cern.ch/e/PyHEP2022
```

Reply:
Ah ha, just to make sure I read :D.

Reply:
Ah, but now I realise that we don't have a URL yet - shall we create a stub agenda to be able to link to?

Reply:
Let me do it. It is on my list anyway ... Give me a sec ...

Reply:
The basic Indico and short link now exist :-). You're good to go.